pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -871,7 +872,14 @@ func NewVirtualMachineCloneCrd() (*extv1.CustomResourceDefinition, error) {
 		Group: clonev1alpha1.VirtualMachineCloneKind.Group,
 		Versions: []extv1.CustomResourceDefinitionVersion{
 			{
-				Name:    clonev1alpha1.SchemeGroupVersion.Version,
+				Name:               clonev1alpha1.SchemeGroupVersion.Version,
+				Served:             true,
+				Storage:            false,
+				Deprecated:         true,
+				DeprecationWarning: pointer.P("clone.kubevirt.io/v1alpha1 VirtualMachineClone is now deprecated and will be removed in v1."),
+			},
+			{
+				Name:    clonev1beta1.SchemeGroupVersion.Version,
```

Comment:
nit: I'ts probably  default value, i can't find it in the docs but with other objects we add:

```
		Conversion: &extv1.CustomResourceConversion{
			Strategy: extv1.NoneConverter,
		},
```

Reply:
Thanks for catching that!
I've verified that it is [the default](https://github.com/kubernetes/kubernetes/blob/3ec97a445f036a38bfec6291dee661954138bac9/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/defaults.go#L42-L54) value:
```go
func SetDefaults_CustomResourceDefinitionSpec(obj *CustomResourceDefinitionSpec) {
	if len(obj.Names.Singular) == 0 {
		obj.Names.Singular = strings.ToLower(obj.Names.Kind)
	}
	if len(obj.Names.ListKind) == 0 && len(obj.Names.Kind) > 0 {
		obj.Names.ListKind = obj.Names.Kind + "List"
	}
	if obj.Conversion == nil {
		obj.Conversion = &CustomResourceConversion{
			Strategy: NoneConverter,
		}
	}
}
```